www/frontends/compiler_gym/src/assets/scss/custom.scss

```diff
@@ -1,5 +1,6 @@
 @import "./variables.scss";
```

Comment:
Every source files needs the Facebook copyright / MIT license header. Some of the scss and js files don't have them, could you please add them (I realize I forgot to do that for demo_api.py!)

Reply:
Noted!